resources/views/admin/book/index.blade.php

```diff
@@ -48,16 +48,35 @@
                                     @foreach($list as $item)
                                     <tr>
                                         <td>{{$i++}}</td>
-                                        <td><a href="{{url('book/'.$item->id.'/edit')}}">{{ $item->name }}</a></td>
+                                        <td><a href="{{url('book/'.$item->id.'/view')}}">{{ $item->name }}</a></td>
                                         <td>{{ $item->category->name }}</td>
                                         <td>{{ $item->author }}</td>
                                         <td>{{ $item->quantity }}</td>
-                                        <!-- <td>{{ $item->publish_year }}</td> -->
                                         <th>{{ date('d/m/Y', strtotime($item->publish_year)) }}</th>
```

Comment:
[rank B] we should be defined format for datetime, number, currency commonly. Please put it on config and re-use. Please imagine in case we need to change datetime format.